packages/strapi-helper-plugin/lib/src/components/InputNumberWithErrors/index.js

```diff
@@ -164,7 +164,7 @@ InputNumberWithErrors.defaultProps = {
   style: {},
   tabIndex: '0',
   validations: {},
-  value: 0,
+  value: '0',
```

Comment:
- Can you please set the default value to `null` instead of `'0'`?

- In the same file, please set the `<InputNumber />` props to `value={value && Number(value)}` instead of the actual `value={value}` props. line 122

- And for the propTypes, set `value: PropTypes.number` instead of `value: PropTypes.oneOfType([PropTypes.string, PropTypes.number])`. line 212